keyboards/40percentclub/25/keymaps/default/keymap.c

```diff
@@ -0,0 +1,91 @@
+/* Copyright 2018 REPLACE_WITH_YOUR_NAME
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _LOWER  1
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  /* Qwerty
+   * ,---------------------------------------------------------------------.
+   * |   1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  |
+   * |------+------+------+------+------+------+------+------+------+------|
+   * |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  |
+   * |------+------+------+------+-------------+------+------+------+------|
+   * |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  | Bksp |
+   * |------+------+------+------+------|------+------+------+------+------|
+   * |   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |Enter |
+   * |------+------+------+------+------+------+------+------+------+------|
+   * | Ctrl | GUI  | Alt  |Lower |Space | Shift| Left | Down |  Up  |Right |
+   * `---------------------------------------------------------------------'
+   */
+  [_QWERTY] = LAYOUT_split(
```

Comment:
It would be better to be named: 
```suggestion
  [_QWERTY] = LAYOUT_ortho_5x10(
```